rust/src/mime/mime.rs

```diff
@@ -0,0 +1,580 @@
+/* Copyright (C) 2021 Open Information Security Foundation
```

Comment:
2023

Reply:
I had understood that this should be the file creation date (already discussed in a previous iteration of the PR).
Should this be the merge date ?